docs/how-to-guides/develop-a-tool/add-a-tool-icon.md

```diff
@@ -1,4 +1,4 @@
-# Adding a Tool Icon
+# Adding a tool icon
```

Comment:
Keep each word capitalized for title.

Reply:
Please only cap the first word following other places: 
![image](https://github.com/microsoft/promptflow/assets/47586720/eae32b91-6d4c-4154-bfa1-67f34ddb09f8)